src/modules/RA/Picture/components/PictureFields.js

```diff
@@ -47,6 +49,16 @@ const useStyles = makeStyles({
       height: 'auto',
       maxHeight: 'inherit',
     },
+    '& .refusal': {
+      position: 'absolute',
+      bottom: '1em',
+      width: '100%',
+      padding: '2em',
```

Comment:
Maybe this element will go outer the width => 100% + 2em + 2em.
The safe workaround in absolute element is to not set a `width: 100%` but `right: 0; left: 0` so you can set the padding you want without side effect

Reply:
Done

Reply:
Almost, you kept the width :D

Reply:
Done